jcl/j9modules.xml

```diff
@@ -60,11 +61,13 @@
 				<removeexportsoropens action="exports" package="com.ibm.le.conditionhandling" />
 				<replaceexportsoropens action="exports" module.old="com.ibm.dtfj" module.new="" />
 				<replaceexportsoropens action="exports" module.old="com.ibm.dtfjview" module.new="" />
+				<removeprovides interface="com.ibm.gpu.spi.GPUAssist.Provider" />
 
 				<delete verbose="true">
 					<fileset dir="${module.dir}/classes">
 						<include name="com/ibm/cuda/**" />
 						<include name="com/ibm/dataaccess/*" />
+						<include name="com/ibm/gpu/internal/*" />
```

Comment:
This is redundant - it's covered by the next line.

Reply:
Sorry, my mistake. We need to keep the 'spi' subdirectory and 'gpu/*' doesn't match subdirectories.